### DISCUSSION TITLE
IDC：聯發科撂倒高通，稱霸美 Andrioid 手機晶片市場

### DISCUSSION THREAD

**Discussion by pekoranoinu** [1101-GS4719-C/Python-Community-GS4719-C#393] [科技新知]
臺灣NO1聯發科超越了美國的高通，我們小小的國家也能夠在國際上有一定的競爭力!
https://technews.tw/2022/03/02/mediatek-might-have-overtaken-qualcomm-in-us-android-marketshare/

**Comment by JSHT** [comment]
MTK 真的猛，新 CPU 工程樣機，天璣 9000 性能贏過 8 gen 1，功耗還略低。天璣 8100 性能跟 8 gen 1 相近，功耗還低不少。
期待有產品正式上市的時候，表現會怎麼樣。
下面是 極客灣 介紹 MTK 新處理器的影片
[天璣 9000](https://www.youtube.com/watch?v=1ves1M4Ai-I)
[天璣 8100](https://www.youtube.com/watch?v=mq0BPkL1ISQ)

**Reply by chia-kai-chang** [reply]
看極客灣~ 有料~

**Comment by xkllkx** [comment]
這幾年，高通原本的市場份額都被聯發科、AMD、Nvidia吃掉了，發哥就是強。
